src/reader/transformers/to-spec.ts

```diff
@@ -25,21 +25,12 @@ function captureAnnotations(node: Unist.Node | undefined): SMdast.IAnnotations {
   return {};
 }
 
-function processBlockquote(node: Mdast.IBlockquote, anno: SMdast.IAnnotations): SMdast.IBlockquote {
-  return {
-    type: 'blockquote',
-    annotations: {
-      ...anno,
-    },
-    children: node.children,
-  };
-}
-
-function processNode(node: Unist.Node, anno: SMdast.IAnnotations | null): Unist.Node {
-  const { type } = node;
-
-  if (type === 'blockquote' && anno) {
-    return processBlockquote(node as Mdast.IBlockquote, anno);
+function processNode(node: Unist.Node, annotations: SMdast.IAnnotations | null): Unist.Node {
```

Comment:
I don't know how much work this is, but I think undefined would be a better choice than null. e.g. `annotations?: SMdast.IAnnotations`.

Already ran into a bug because tried to use default assignment somewhere, and due to null usage here default assignment did not work.

Reply:
Easy peasy change :)